server/config/default.json

```diff
@@ -40,7 +40,7 @@
     "apiSecurity": false
   },
   "auth": {
-    "type": "local",
+    "registration": "disabled",
```

Comment:
Make it boolean value `true` or `false`, since it has only 2 options, and can't be "misstyped"

Reply:
Good call i'll sort this one later